AlphaGo/training/supervised_policy_trainer.py

```diff
@@ -85,6 +85,57 @@ def on_epoch_end(self, epoch, logs={}):
 }
 
 
+def load_indices_from_file(symmetries, shuffle_file):
+    # load indices from file
+    with open(shuffle_file, "r") as f:
+        indices = np.load(f)
+
+    return indices
+
+
+def set_indices(args, n_total_data_size, symmetries, shuffle_file_train, shuffle_file_val):
+
+    # create array with all position, symmetry combinations
```

Comment:
Use a more descriptive comment here. For example:

`# Create an array with a unique row for each combination of a training example and a symmetry. shuffle_indices[i][0] is an index into training examples, and shuffle_indices[i][1] is the index (from 0 to 7) of the symmetry transformation to apply.`